src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+public class Main {
+
+    public static void main(String[] args) {
+        Figure[] figures = new Figure[6];
+        FigureSupplier figureSupplier = new FigureSupplier();
+        Figure defaultFigure = figureSupplier.getDefaultFigure();
+        figures[0] = defaultFigure;
+        System.out.println(defaultFigure.draw());
+        for (int i = 1; i < 6; i++) {
```

Comment:
Use figures.length instead of hardcoded value (6) in the for loop condition.

Also your task was to create 3 random and 3 default figures